link_libsodium.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+readonly CWD=$PWD
+readonly LIBSODIUM_VERSION="libsodium-1.0.17"
+
+test -d tmp/libsodium || mkdir -p tmp/libsodium
+
+cd tmp/libsodium
+
+curl -sL https://download.libsodium.org/libsodium/releases/$LIBSODIUM_VERSION.tar.gz -o $LIBSODIUM_VERSION.tar.gz
+tar xfz $LIBSODIUM_VERSION.tar.gz --strip-components=1
+
+./configure --prefix $PWD
+make && make check && make install
```

Comment:
Don't invoke make commands from the script which is invoked from Makefile.

Reply:
This is necessary to install the [libsodium](https://libsodium.gitbook.io/doc/installation#compilation-on-unix-like-systems).